example/src/App.test.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import App from './App'
+import { createRoot } from 'react-dom/client';
+
 
 it('renders without crashing', () => {
   const div = document.createElement('div')
-  ReactDOM.render(<App />, div)
+  const root = createRoot(div);
+  root.render(<App />)
```

Comment:
Does this actually maintain backward compatibility? Specifically - does React 16/17 _have_ a `createRoot`? Or does this just fix it for React 18?

Reply:
This is just an example. In your app you have to use the API that works with your React version